test/examples/ieee14bus.jl

```diff
@@ -3,10 +3,11 @@ using JLD2
 using PowerDynamics
 
 # read grid data from json file
-pg_read = read_powergrid("../examples/ieee14bus/grid.json", Json)
+ieee14path = joinpath(dirname(pwd()),"examples/ieee14bus/")
```

Comment:
```suggestion
ieee14path = joinpath(dirname(pwd()), "examples", "ieee14bus")
```
maybe even this to make sure there is no problem with `/` and `\` on different platforms?